app/controllers/idv/review_controller.rb

```diff
@@ -62,6 +65,10 @@ def create
 
     private
 
+    def log_reproof_event
+      irs_attempts_api_tracker.idv_reproof
+    end
```

Comment:
this is such a simple method, would it be easier to inline this?

Reply:
I don't feel too strongly, but I think I have a slight preference for keeping it a separate method. It just feel slightly more expressive.